### PR TITLE
[dualtor] Add support to setup netns topo

### PR DESCRIPTION
**Pull request by lolyu**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
To support `active-active` dualtor network namespace setup.

#### How did you do it?
1. Add new Ansible module `mux_cable_facts` to gather cable type, server ip and soc ip from topology defined.
2. Use `mux_cable_facts` for minigraph gen/deloy.
3. Support network namespace interface ip assignment and route setup.

#### How did you verify/test it?
Run `add-topo` and `remove-topo` for dualtor-mixed topology.

### CODE REVIEW

**Comment by lolyu** [comment]
@wangxin, could you please take a reivew?

File: ansible/roles/vm_set/library/vm_topology.py [hunk 1]
```diff
@@ -446,6 +475,25 @@ def add_br_if_to_docker(self, bridge, ext_if, int_if):
 
         VMTopology.iface_up(int_if, pid=self.pid)
 
+    def add_br_if_to_netns(self, bridge, ext_if, int_if):
```
Comment (wangxin) [inline ansible/roles/vm_set/library/vm_topology.py:478]: I think the function name is a little bit misleading. Either use a better name or add some docstring to explain what is done by this function?

**Review by wangxin — COMMENTED** [review]
Can you also help remove below misleading comment in the vm_topology.py file?
This is not a TODO in `vm-topology.py`. It will be taken care by the new simulator.

`# TODO: open-flow configuration for ovs-bridge simulating server smart NIC`

**Comment by lolyu** [comment]
@wangxin, updated, please take a review, thanks!

**Review by wangxin — APPROVED** [review]

**Pull request merged**